llvm/test/Transforms/HotColdSplit/outline-inner-region-stacktoocomplex.ll

```diff
@@ -0,0 +1,51 @@
+; RUN: opt -S -passes=hotcoldsplit -hotcoldsplit-max-params=1 < %s | FileCheck %s
+
+target datalayout = "E-m:a-p:32:32-i64:64-n32"
+target triple = "powerpc64-ibm-aix7.2.0.0"
```

Comment:
Is this triple needed? If it is, the test needs to be marked as requiring the PowerPC backend, otherwise it will fail when it is not built.

Reply:
Yes I retained it since I modified existing test with some changes. will remove.